src/components/Article.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from "react";
+import { getArticles, getArticlesById } from "../api";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+import { Comments } from "./Comments";
+
+export const Article = () => {
+  const params = useParams();
+  const [article, setArticle] = useState(null);
+  const [loadingArticle, setLoadingArticle] = useState(false);
+
+  useEffect(() => {
+    setLoadingArticle(true);
+    getArticlesById(params.article_id).then((data) => {
+      setArticle(data.articles);
+      setLoadingArticle(false);
+    });
+  }, [params.article_id]);
+
+  if (!article) {
```

Comment:
What was the intention with this block of code? (Genuinely asking)

Reply:
If there was no article of that id it should return null? maybe this can be covered in the error block?